### PR TITLE
luci-theme-openwrt-2020: fix anchor decorations

### PR DESCRIPTION
**Pull request by stangri**
* add the decorations for when the links are included in the field descriptions

Thanks @dibdot!

### CODE REVIEW

File: themes/luci-theme-openwrt-2020/htdocs/luci-static/openwrt2020/cascade.css [hunk 1]
```diff
@@ -1123,6 +1123,19 @@ textarea {
 	padding: .2em 0;
 }
 
+.cbi-value-description a {
+  color: var(--main-bright-color);
+  text-decoration: none;
+  line-height: inherit;
```
Comment (stokito) [inline themes/luci-theme-openwrt-2020/htdocs/luci-static/openwrt2020/cascade.css:1129]: I'm not a CSS expert but the inherit seems like a hack. Maybe we can avoid it?

https://developer.mozilla.org/en-US/docs/Web/CSS/inherit

**Review by stokito — APPROVED** [review]

**Comment by dibdot** [comment]
@stangri I've tested the change on my current snapshot. Could you please change your PR as follows:
```
.cbi-map-descr a,
.cbi-value-description a {
  color: var(--main-bright-color);
  text-decoration: none;
  outline: 0;
}
.cbi-map-descr a:hover,
.cbi-value-description a:hover {
  color: var(--main-bright-color);
  text-decoration: underline;
  outline: 0;
}
```

The '.cbi-map-descr' is needed to fix the anchor decoration for adblock, banIP and travelmate as well ... :-)  and it seems  that both inherit lines are needless.

Thanks for your contribution!

**Comment by stangri** [comment]
@stokito, not sure what you suggested as a fix, but since you thumbed up @dibdot's suggestion I'm guessing you're OK with that.

Feel free to merge and then I'll cherry-pick the commit and merge it to stable.

**Comment by stangri** [comment]
@stokito ping?

**Comment by stokito** [comment]
Sure, I'm ok.
@bryanforbes you are added the theme but not mentioned as a maintaner for theme.
Could you become a maintainer for it?
If yes, then please approve the PR

**Review by stokito — APPROVED** [review]

**Comment by bryanforbes** [comment]
@stokito I didn't add this theme. I think I submitted a fix a long time ago, but I didn't add the theme.

**Pull request merged**